src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -30,6 +30,7 @@ IqPolicyEvaluation.FailOnNetwork=Fail build when unable to communicate with IQ S
 IqPolicyEvaluation.JobSpecificCredentials=Use job specific credentials
 
 IqPolicyEvaluation.NoIqServersConfigured=No IQ Server configured.
+IqPolicyEvaluation.ResourceNotFound=Possible version mismatch.  Please check if the IQ Server is compatible with this plugin.
```

Comment:
Minor:  Maybe reword to:

Possible version mismatch.  Please check if the IQ Server is compatible with this plugin version.

Reply:
> Maybe reword to

changed in 02dd0fa